src/main/java/mods/eln/gui/GuiHelper.java

```diff
@@ -75,6 +75,13 @@ public GuiVerticalProgressBar newGuiVerticalProgressBar(int x, int y, int width,
         return o;
     }
 
+    public GuiVerticalCustomValuesBar newGuiVerticalCustomValuesBar(int x, int y, int width, int height, Float[] values) {
```

Comment:
There's a limited amount of documentation, unfortunately. I've been browsing the code a bit, and talking to other devs. But: https://github.com/TeamWizardry/LibrarianLib/wiki/GUI-API-Home

1.10 only thus far. There is an 1.7 version, but it's different enough that it isn't worth it.